### DISCUSSION TITLE
Request From the Field/User:  Enable features in the Question Set Editor editor and  QUML Player

### DISCUSSION THREAD

**Discussion by vinod303** [Sunbird-inQuiry/Community#100] [Feature Requests]
Following  feature request  has been made by and existing Sunbird User for Question set editor and Player

![image](https://github.com/Sunbird-inQuiry/Community/assets/15699608/8af303ce-975f-448e-a803-772a8564338f)

@parthlawate
@PoojaJManjooran

**Comment by PoojaJManjooran** [comment]
Hi @vinod303 - We will analyse this. Meanwhile, are all of these currently required, and is there a priority ranking for the question types?
